src/main/java/Main.java

```diff
@@ -56,18 +55,42 @@ public static void main(String[] args) throws Exception {
                     System.out.println("your role");
                     String inputRole = scanner.nextLine();
                     UserRole createRole = UserRole.valueOf(inputRole.toUpperCase());
-                    new RegisterCommand().register(createUserName, createPassword, createName, createSurname, createRole);
-                    break;
-
-                case CommandConstants.EXIT_CODE:
+                    new RegisterCommand().
+                            register(createUserName, createPassword, createName, createSurname, createRole);
+                }
+                case CommandConstants.EXIT_CODE -> {
                     ExitCommand exitCommand = new ExitCommand();
                     exitCommand.exit();
-                    break;
-
-                case CommandConstants.SHOW_ME_LIST_CODE:
+                }
+                case CommandConstants.SHOW_ME_LIST_CODE -> {
                     PrintUsersCommand printUsersCommand = new PrintUsersCommand();
                     printUsersCommand.print();
-                    break;
+                }
+                case CommandConstants.BAN_USERS -> {
+                    User currentUser = UserSession.currentUser;
```

Comment:
It would be better to retrieve current role here. And then use it in the `if`
You don’t use user in other places. So it will look better